test/src/test/java/hudson/model/AbstractBuildTest2.java

```diff
@@ -71,12 +71,6 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListen
         }
         
         @TestExtension("reportErrorShouldNotFailForNonPublisherClass")
-        public static class DescriptorImpl extends JobPropertyDescriptor {
-
-            @Override
-            public String getDisplayName() {
-                return "Always throws exception in perform()";
-            }  
-        }
+        public static class DescriptorImpl extends JobPropertyDescriptor {}
```

Comment:
Not sure if it *really* matters at all, but many of these will just end up having the same name of "DescriptorImpl"